### PR TITLE
fix: update catalog with latest ai-lab-recipes

### LINKED ISSUE #2120: Update catalog to the latest version
### Is your enhancement related to a problem? Please describe

Use latest version of the ai-lab recipes

### Describe the solution you'd like

Update references

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by jeffmaury**
Fixes #2120

### What does this PR do?

Update the catalog with latest released version of recipes

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

#2120 

### How to test this PR?

Run recipes

### CODE REVIEW

**Review by benoitf — APPROVED** [review]
LGTM but I have not tested all the recipes

**Comment by feloy** [comment]
I get an error with the summarizer:

summarizer-bug

**Comment by jeffmaury** [comment]
> I get an error with the summarizer (with an .MD file, it worked with a .TXT file):
> 
> summarizer-bug

I don't think this is related to this PR I think the application only supports .TXT and .PDF files

**Comment by feloy** [comment]
> > I get an error with the summarizer (with an .MD file, it worked with a .TXT file):
> > summarizer-bug
> 
> I don't think this is related to this PR I think the application only supports .TXT and .PDF files

The problem I can see is that the interface is displaying that the supported formats are TXT, PDF and MD.

I proposed a patch on the ai-lab-recipes repo to support MD files: https://github.com/containers/ai-lab-recipes/pull/815

**Review by gastoner — APPROVED** [review]
Codewise LGTM

**Comment by feloy** [comment]
> 
> PR updated can you retry ?

This works now, thanks

**Comment by jeffmaury** [comment]
> For the RAG, I get this error:
> 
> rag_error

Let me check because it was supposed to be fixed

**Comment by jeffmaury** [comment]
> > For the RAG, I get this error:
> > rag_error
> 
> Let me check because it was supposed to be fixed

Works on my machine :)

**Comment by feloy** [comment]
> > > For the RAG, I get this error:
> > > rag_error
> > 
> > 
> > Let me check because it was supposed to be fixed
> 
> Works on my machine :)

Works on my machine now :) I don't know what is different from the previous test

File: packages/backend/src/assets/ai.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
       "description": "This is a Quarkus-based chat demo application.",
       "name": "Java-based ChatBot (Quarkus)",
       "repository": "https://github.com/containers/ai-lab-recipes",
-      "ref": "a7b03d7bd025e009a03147391a00cb1ee472ad3f",
+      "ref": "v1.3.3",
       "icon": "natural-language-processing",
       "categories": ["natural-language-processing"],
       "basedir": "recipes/natural_language_processing/chatbot-java",
```
Comment (feloy) [inline packages/backend/src/assets/ai.json:80]:
```suggestion
      "basedir": "recipes/natural_language_processing/chatbot-java-quarkus",
```

**Review by feloy — CHANGES_REQUESTED** [review]

**Review by feloy — APPROVED** [review]
tested all recipes, all are working for me

https://github.com/containers/ai-lab-recipes/pull/815 has been merged. Is it worth waiting for a new release to include it?

**Comment by jeffmaury** [comment]
> tested all recipes, all are working for me
> 
> [containers/ai-lab-recipes#815](https://github.com/containers/ai-lab-recipes/pull/815) has been merged. Is it worth waiting for a new release to include it?

I'm still fighting for the release as I can't make the Quarkus recipe to work so I would delay as the goal of this release is to fix the errors on existing recipes

**Comment by feloy** [comment]
Is the tag `1.3.3.1` already done on the `ai-lab-recipes`? I cannot see it

**Comment by jeffmaury** [comment]
> Is the tag `1.3.3.1` already done on the `ai-lab-recipes`? I cannot see it

https://github.com/containers/ai-lab-recipes/releases/tag/v1.3.0.1

File: packages/backend/src/assets/ai.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
       "description": "This is a Streamlit chat demo application.",
       "name": "ChatBot",
       "repository": "https://github.com/containers/ai-lab-recipes",
-      "ref": "v1.3.0",
+      "ref": "v1.3.3.1",
```
Comment (feloy) [inline packages/backend/src/assets/ai.json:9]:
```suggestion
      "ref": "v1.3.0.1",
```

File: packages/backend/src/assets/ai.json [hunk 3]
```diff
@@ -23,7 +23,7 @@
       "description": "This is a Streamlit demo application for summarizing text.",
       "name": "Summarizer",
       "repository": "https://github.com/containers/ai-lab-recipes",
-      "ref": "v1.3.0",
+      "ref": "v1.3.3.1",
```
Comment (feloy) [inline packages/backend/src/assets/ai.json:26]:
```suggestion
      "ref": "v1.3.0.1",
```

File: packages/backend/src/assets/ai.json [hunk 4]
```diff
@@ -40,7 +40,7 @@
       "description": "This is a code-generation demo application.",
       "name": "Code Generation",
       "repository": "https://github.com/containers/ai-lab-recipes",
-      "ref": "v1.3.0",
+      "ref": "v1.3.3.1",
```
Comment (feloy) [inline packages/backend/src/assets/ai.json:43]:
```suggestion
      "ref": "v1.3.0.1",
```

File: packages/backend/src/assets/ai.json [hunk 5]
```diff
@@ -57,7 +57,7 @@
       "description": "This is a Streamlit RAG enabled chat demo application.",
       "name": "RAG Chatbot",
       "repository": "https://github.com/containers/ai-lab-recipes",
-      "ref": "v1.3.0",
+      "ref": "v1.3.3.1",
```
Comment (feloy) [inline packages/backend/src/assets/ai.json:60]:
```suggestion
      "ref": "v1.3.0.1",
```

File: packages/backend/src/assets/ai.json (hunk 2 above)
Comment (jeffmaury) [reply packages/backend/src/assets/ai.json:9]: Tag renamed for better consistency

File: packages/backend/src/assets/ai.json (hunk 3 above)
Comment (jeffmaury) [reply packages/backend/src/assets/ai.json:26]: Tag renamed for better consistency

File: packages/backend/src/assets/ai.json (hunk 4 above)
Comment (jeffmaury) [reply packages/backend/src/assets/ai.json:43]: Tag renamed for better consistency

File: packages/backend/src/assets/ai.json [hunk 6]
```diff
@@ -74,10 +74,10 @@
       "description": "This is a Quarkus-based chat demo application.",
       "name": "Java-based ChatBot (Quarkus)",
       "repository": "https://github.com/containers/ai-lab-recipes",
-      "ref": "a7b03d7bd025e009a03147391a00cb1ee472ad3f",
+      "ref": "v1.3.3.1",
```
[the remark this answers is not in the document]
Comment (jeffmaury) [reply packages/backend/src/assets/ai.json:77]: Tag renamed for better consistency

File: packages/backend/src/assets/ai.json [hunk 7]
```diff
@@ -91,7 +91,7 @@
       "description": "This is a Streamlit function calling enabled demo application.",
       "name": "Function calling",
       "repository": "https://github.com/containers/ai-lab-recipes",
-      "ref": "v1.3.0",
+      "ref": "v1.3.3.1",
```
[the remark this answers is not in the document]
Comment (jeffmaury) [reply packages/backend/src/assets/ai.json:94]: Tag renamed for better consistency

File: packages/backend/src/assets/ai.json [hunk 8]
```diff
@@ -104,7 +104,7 @@
       "description": "This is an audio to text demo application.",
       "name": "Audio to Text",
       "repository": "https://github.com/containers/ai-lab-recipes",
-      "ref": "v1.3.0",
+      "ref": "v1.3.3.1",
```
[the remark this answers is not in the document]
Comment (jeffmaury) [reply packages/backend/src/assets/ai.json:107]: Tag renamed for better consistency

File: packages/backend/src/assets/ai.json [hunk 9]
```diff
@@ -117,7 +117,7 @@
       "description": "This is an object detection demo application.",
       "name": "Object Detection",
       "repository": "https://github.com/containers/ai-lab-recipes",
-      "ref": "v1.3.0",
+      "ref": "v1.3.3.1",
```
[the remark this answers is not in the document]
Comment (jeffmaury) [reply packages/backend/src/assets/ai.json:120]: Tag renamed for better consistency

File: packages/backend/src/assets/ai.json (hunk 5 above)
Comment (jeffmaury) [reply packages/backend/src/assets/ai.json:60]: Tag renamed for better consistency

**Comment by feloy** [comment]
LGTM

**Pull request merged**